pkgs/games/prismlauncher/default.nix

```diff
@@ -46,7 +46,10 @@ stdenv.mkDerivation rec {
 
   hardeningEnable = [ "pie" ];
 
-  cmakeFlags = lib.optionals (msaClientID != null) [ "-DLauncher_MSA_CLIENT_ID=${msaClientID}" ]
+  cmakeFlags = [
+    # downstream branding
+    "-DLauncher_BUILD_PLATFORM=nixpkgs"
+  ] ++ lib.optionals (msaClientID != null) [ "-DLauncher_MSA_CLIENT_ID=${msaClientID}" ]
```

Comment:
can I query the

> no packaging changes

bit please? I think from the comment this is only doing branding but I want to make sure it's nothing more than that

Reply:
Relevant pull request: https://github.com/PrismLauncher/PrismLauncher/pull/1276

Reply:
thanks, I think this is fine then, thank you very much!